Tests/Brokerages/Paper/PaperBrokerageTests.cs

```diff
@@ -117,8 +116,8 @@ public void AppliesDividendsOnce()
             var brokerage = new PaperBrokerage(algorithm, job);
 
             // initialize results and transactions
-            results.Initialize(job, new EventMessagingHandler(), new Api.Api(), new BrokerageSetupHandler(), transactions);
-            results.SetAlgorithm(algorithm);
+            results.Initialize(job, new EventMessagingHandler(), new Api.Api(), transactions);
+            results.SetAlgorithm(algorithm, algorithm.Portfolio.TotalPortfolioValue);
```

Comment:
I like this, much better than exposing the entire `ISetupHandler` just to grab initial TPV 👍